### DISCUSSION TITLE
Costings justification

### DISCUSSION THREAD

**Discussion by gavpop** [magicbug/Cloudlog#3171] [Q&A]
Hi - I'm very interested in Cloudlog and leaning towards a subscription, to save me hassle of servers etc.
Even though £48 isn't going to break the bank, I still want to justify the cost.
I currently pay £24 a year for QRZ.com to allow me auto-uploading of QSL data from Log4OM.
If I purchased Cloudlog subscription, could I dispense of the £24 a year for QRZ.com and still get auto-uploading of QSL data to QRZ?

TIA - Gav, M7GDU.

**Comment by magicbug** [comment]
The price is purely based on what it costs to provide a quality service, there's plenty other suitable hosting providers available and at a lower cost.

Cloudlog can upload to QRZ.com I'm not sure whether you'd still need a subscription to use QRZ Logbook you'd need to check that.
